### PR TITLE
updates

### PR DESCRIPTION
**Pull request by tbradyravis**

### CODE REVIEW

File: ring/gallery.html [hunk 1]
```diff
@@ -28,9 +29,23 @@
             <br>
             <br>
             <span class="c1">To view an image, click on its thumbnail.
+            </span>
             <br>
             <br>
-            <a href="1.jpg"><img src="1small.jpg" height="107" width="66" alt="no alt text"></a> <a href="2.jpg"><img src="2small.jpg" height="104" width="69" alt="no alt text"></a> <a href="3.jpg"><img src="3small.jpg" height="78" width="98" alt="no alt text"></a> <a href="4.jpg"><img src="4small.jpg" height="104" width="69" alt="no alt text"></a> <a href="5.jpg"><img src="5small.jpg" height="89" width="109" alt="no alt text"></a> <a href="6.jpg"><img src="6small.jpg" height="120" width="72" alt="no alt text"></a></span>
+            <a href="1.jpg"><img src="1small.jpg" height="107" width="66" alt="no alt text"></a>
```
Comment (gemohandy) [inline ring/gallery.html]: These images are not very descriptively named. (1.jpg through 6.jpg? Really?) You should fix that.

File: athorn/Gallery.html [hunk 2]
```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <title> Gallery </title>
+  <link href="css/style.css" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css?family=Reem+Kufi" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css?family=Lemonada" rel="stylesheet">
+  <link href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet" integrity="sha384-wvfXpqpZZVQGK6TAh5PVlGOfQNHSoD2xbE+QkPxCAFlNEevoEH3Sl0sibVcOQVnN" crossorigin="anonymous">
```
Comment (gemohandy) [inline athorn/Gallery.html]: Should Bootstrap and Font-Awesome be downloaded locally? I am aware that it is done on at least one other new site, and is done to make the page load faster (I think).

File: athorn/css/style.css [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+  margin-top: 100;
+  border: 15px solid #04102A;
+}
+
+ul{
+  margin: 0;
+  padding: 0;
+}
+
+a{
```
Comment (gemohandy) [inline athorn/css/style.css]: Brett does not want global style on links. Add a class.

File: athorn/css/style.css [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+
+}
+
+.image{
+  width: 400px;
+  margin-top: 100;
+  border: 15px solid #04102A;
+}
+
+ul{
```
Comment (gemohandy) [inline athorn/css/style.css]: Brett does not want global style on links. Add a class.

File: athorn/index.html [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+
+    </div>
+
+    <div class ="info">
+
+      <h1 class="titleinfo"> About Anthony Thorn </h1>
+      <p class="bio"> Anthony Thorn, born Arthur Goldman, is a prominent Saskatchewan artist from Regina born in 1927.
+        <br> Thorn began painting in 1948 shortly after graduating from the University of Saskatchewan with a Bachelor of Arts.
+        <br> with a Bachelor of Arts. He had nearly 1200 original wokrks in his collection using the mediums of drawing, oil
+        <br> painting, sculpture, and watercolour. Thorn was a founding member of the Saskatchewan Society of Artists and has
```
Comment (gemohandy) [inline athorn/index.html]: May I ask about the line formating? It seems a bit weird.

File: ring/frankensteingallery.html [hunk 6]
```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+    <meta name="GENERATOR" content="Mozilla/4.73 [en] (Windows NT 5.0; U) [Netscape]">
+    <title> Frankenstein Gallery </title>
+    <style type="text/css">
```
Comment (gemohandy) [inline ring/frankensteingallery.html]: Should this be in an external style sheet?

File: athorn/Gallery.html (hunk 2 above)
Comment (BmanDesignsCanada) [inline athorn/Gallery.html]: Using a major CDN like maxcdn is fine.

File: athorn/index.html (hunk 5 above)
Comment (BmanDesignsCanada) [inline athorn/index.html]: <br/> instead of <br>

File: ring/frankensteingallery.html (hunk 6 above)
Comment (BmanDesignsCanada) [inline ring/frankensteingallery.html]: Yes, but I will accept it for now

File: athorn/index.html (hunk 5 above)
Comment (gemohandy) [inline athorn/index.html]: I was more asking about the fact that there are so many line breaks. Why are they there? Doesn't the text automatically wrap?

File: athorn/index.html [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+        <br> fellow artist, after writing an insightful review of an exhibition of his work in 1990.
+
+
+
+     </p>
+
+    </div>
+
+    <footer class="footer"> Created by Travis Brady, 2017
+      <br> Copyright © 2017 Anthony Thorn
```
Comment (BmanDesignsCanada) [inline athorn/index.html]: Should be `&copy;`

Comment (gemohandy) [inline athorn/index.html]: Make sure to change this on all the pages.

File: athorn/Gallery.html [hunk 8]
```diff
@@ -0,0 +1,67 @@
...
+  <link href="https://fonts.googleapis.com/css?family=Lemonada" rel="stylesheet">
+  <link rel="stylesheet" href="./css/bootstrap.min.css">
+</head>
+
+<body>
+
+  <div class="nav">
+    <ul>
+      <li class="nav-link"><a href="index.html"> Home </a></li>
+      <li class="nav-link"><a href="Gallery.html"> Gallery </a></li>
```
Comment (BmanDesignsCanada) [inline athorn/Gallery.html]: Lowercase file names

**Comment by gemohandy** [comment]
Remaining Problems:
- [ ]Roger Ing Image Names
- [ ]Slashes in Line Breaks

File: athorn/Gallery.html [hunk 9]
```diff
@@ -14,7 +14,7 @@
   <div class="nav">
     <ul>
       <li class="nav-link"><a href="index.html"> Home </a></li>
-      <li class="nav-link"><a href="Gallery.html"> Gallery </a></li>
+      <li class="nav-link"><a href="gallery.html"> Gallery </a></li>
```
Comment (gemohandy) [inline athorn/Gallery.html]: ...Did you just break this link?

File: athorn/index.html [hunk 10]
```diff
@@ -13,7 +13,7 @@
     <div class ="nav">
       <ul>
         <li class ="nav-link"><a href="index.html"> Home </a></li>
-        <li class ="nav-link"><a href="Gallery.html"> Gallery </a></li>
+        <li class ="nav-link"><a href="gallery.html"> Gallery </a></li>
```
Comment (gemohandy) [inline athorn/index.html]: Seriously, you never renamed Gallery.html to gallery.html.

**Pull request merged**